### PR TITLE
#170471116 Create app top area

### PR DESCRIPTION
**Pull request by Bradogilo**
#### What does this PR do?

This PR creates the 1990 designs social app top area

#### Task to be completed

- add cover image
- add profile image
- add update info button
- add view activity button

#### Relevant Pivotal Tracker Story

#170471116

#### Relevant Screenshots

Screenshot 2019-12-30 at 23 55 42

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -96,3 +102,81 @@ header .search-area {
...
+}
+
+#top-area .profile-info .user-image {
+  padding: 10px;
+  display: inline-block;
+}
+
+#top-area .profile-info .user-image img {
+  border-radius: 50%;
+  /* filter: grayscale(100%); */
```
Comment (DerKip) [inline css/main.css]: remove this commented code

File: css/main.css [hunk 2]
```diff
@@ -96,3 +102,81 @@ header .search-area {
...
+}
+
+#top-area .profile-info .user-details .action-buttons button {
+  margin-left: 0px;
+  margin-right: 20px;
+  width: 45%;
+  text-align: center;
+  text-transform: uppercase;
+  background: rgb(244, 244, 244, 0.2);
+  /* background: rgb(255, 255, 255, 0.1); */
```
Comment (DerKip) [inline css/main.css]: same here

File: css/main.css (hunk 1 above)
Comment (Bradogilo) [reply css/main.css]: Thanks for pointing that out. Let me do that.

File: css/main.css (hunk 2 above)
Comment (Bradogilo) [reply css/main.css]: On it.

File: css/main.css [hunk 3]
```diff
@@ -96,3 +102,81 @@ header .search-area {
...
+
+#top-area .profile-info .user-image img {
+  border-radius: 50%;
+  /* filter: grayscale(100%); */
+}
+
+#top-area .profile-info .user-details {
+  display: inline-block;
+  vertical-align: top;
+  margin-left: 25px;
```
Comment (DerKip) [inline css/main.css:143]: maybe consider increasing the margin-left for it to align with the logo on the header

Screenshot 2019-12-31 at 00 26 28

Comment (Bradogilo) [reply css/main.css:143]: Thanks, on it.

File: css/main.css (hunk 1 above)
Comment (Bradogilo) [reply css/main.css]: Done

File: css/main.css (hunk 2 above)
Comment (Bradogilo) [reply css/main.css]: Done

File: css/main.css (hunk 3 above)
Comment (Bradogilo) [reply css/main.css:143]: Done

**Comment by DerKip** [comment]
LGTM

**Pull request merged**